projects/packages/jetpack-mu-wpcom/src/features/wpcom-global-styles/index.php

```diff
@@ -227,6 +227,15 @@ function wpcom_global_styles_enqueue_assets() {
 		$asset_file['version'] ?? filemtime( Jetpack_Mu_Wpcom::BASE_DIR . 'build/wpcom-global-styles-frontend/wpcom-global-styles-frontend.js' ),
 		true
 	);
+	wp_add_inline_script(
+		'wpcom-global-styles-frontend',
+		'const launchBarUserData = ' . wp_json_encode(
+			array(
+				'blogId' => get_current_blog_id(),
```

Comment:
I wonder if you should consider here whether the site is Atomic or not, because the blog ID is determined differently for the launch banner on Atomic sites:

https://github.com/Automattic/jetpack/blob/59e2d931642816fa7b02230dc4c8b0fddf1cddc4/projects/plugins/wpcomsh/private-site/logged-in-banner.php#L190

Reply:
Never mind, just saw its handled a few lines above.